lib/status_code/locales/en.yml

```diff
@@ -403,6 +403,390 @@ en:
       '502': Decline, pick-up card, card issuer decline reason # Отказ от эмитента – необходим захват карты
       '809': Error, system error # Системная ошибка
       '888': Error, invalid session number # Неверный номер сессии
+    bankone:
```

Comment:
`bank_one` would be the right version for `BankOne` class :)
So, use `bank_one`, but update the places where we use the library(wls, payment_page) to correctly pass gateway